.github/workflows/e2e.yml

```diff
@@ -1,12 +1,8 @@
 name: SalesforceCommerceCloud/pwa-kit/e2e
 on:
-  workflow_dispatch:
-  push:
-    branches:
-      - setup-e2e-testing
   schedule:
     # Run every day at 12am (PST) - cron uses UTC times
-    - cron: "0 8 * * *"
+    - cron: "*/5 * * * *"
```

Comment:
I assume this will be reverted before merge?

Reply:
Reverted!